python/lsst/meas/astrom/astrometry_net.cc

```diff
@@ -0,0 +1,598 @@
+            qa->set("meas_astrom*an*best_index*hp", ind->healpix);
+            qa->set("meas_astrom*an*best_index*nside", ind->hpnside);
+            qa->set("meas_astrom*an*best_index*name", std::string(ind->indexname));
+        }
+        if (_solver->have_best_match) {
+            MatchObj* mo = &(_solver->best_match);
+            std::string s = boost::str(boost::format("%i") % mo->star[0]);
+            for (int i=1; i<mo->dimquads; i++)
+                s = s + boost::str(boost::format(", %i") % mo->star[i]);
+            qa->set("meas_astrom*an*best_match*starinds", s);
+            qa->set("meas_astrom*an*best_match*coderr", std::sqrt(mo->code_err));
+            qa->set("meas_astrom*an*best_match*nmatch", mo->nmatch);
+            qa->set("meas_astrom*an*best_match*ndistract", mo->ndistractor);
+            qa->set("meas_astrom*an*best_match*nconflict", mo->nconflict);
+            qa->set("meas_astrom*an*best_match*nfield", mo->nfield);
+            qa->set("meas_astrom*an*best_match*nindex", mo->nindex);
+            qa->set("meas_astrom*an*best_match*nbest", mo->nbest);
+            qa->set("meas_astrom*an*best_match*logodds", mo->logodds);
+            qa->set("meas_astrom*an*best_match*parity", mo->parity ? 0 : 1);
+            qa->set("meas_astrom*an*best_match*nobjs", mo->objs_tried);
+        }
+        return qa;
+    }
+
+    std::shared_ptr<lsst::afw::image::Wcs> getWcs() {
+        MatchObj* match = solver_get_best_match(_solver.get());
+        if (!match)
+            return std::shared_ptr<afw::image::Wcs>();
+        tan_t* wcs = &(match->wcstan);
+
+        afw::geom::Point2D crpix(wcs->crpix[0], wcs->crpix[1]);
+        std::shared_ptr<afw::coord::Coord const> crval
+            (new afw::coord::Coord(wcs->crval[0] * afw::geom::degrees,
+                                 wcs->crval[1] * afw::geom::degrees));
+        return afw::image::makeWcs(*crval, crpix,
+                                 wcs->cd[0][0], wcs->cd[0][1],
+                                 wcs->cd[1][0], wcs->cd[1][1]);
+    }
+
+    bool didSolve() {
+        return solver_did_solve(_solver.get());
+    }
+
+    void run(double cpulimit) {
+        solver_log_params(_solver.get());
+        struct timer_baton tt;
+        if (cpulimit > 0.) {
+            tt.s = _solver.get();
+            tt.timelimit = cpulimit;
+            _solver->userdata = &tt;
+            _solver->timer_callback = timer_callback;
+        }
+        solver_run(_solver.get());
+        if (cpulimit > 0.) {
+            _solver->timer_callback = NULL;
+            _solver->userdata = NULL;
+        }
+    }
+
+    double getQuadSizeLow() {
+        double qlo,qhi;
+        solver_get_quad_size_range_arcsec(_solver.get(), &qlo, &qhi);
+        return qlo;
+    }
+    double getQuadSizeHigh() {
+        double qlo,qhi;
+        solver_get_quad_size_range_arcsec(_solver.get(), &qlo, &qhi);
+        return qhi;
+    }
```

Comment:
I changed my mind. It's just used once in Python so I'll rename it to GetQuadSizeRangeArcsec to match the astrometry_net call.